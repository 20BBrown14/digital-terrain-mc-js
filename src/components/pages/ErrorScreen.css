.error-screen-container {
  position: fixed;
  width: 100%;
  height: 100%;
  display: flex;
  align-items: center;
  top: 0;
  right: 0
}

.error-screen-element {
  margin: 0 auto;
  width: 75%;
  text-align: center;
}