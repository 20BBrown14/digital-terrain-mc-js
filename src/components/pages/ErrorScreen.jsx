import React from 'react';
import { Typography } from 'antd';
import './ErrorScreen.css';
import deadHearts from '../../assets/Dead-hearts.gif';

const ErrorScreen = () => (
  <div className="error-screen-container">
    <div className="error-screen-element">
      <img src={deadHearts} alt="" />
      <Typography.Title>
        Something happened trying to load this page. Please try again later.
      </Typography.Title>
    </div>
  </div>
);

export default ErrorScreen;
