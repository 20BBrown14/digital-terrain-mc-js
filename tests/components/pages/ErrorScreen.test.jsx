import React from 'react';
import ErrorScreen from '../../../src/components/pages/ErrorScreen';

describe('ErrorScreen', () => {
  it('renders a default view', () => {
    const errorScreen = shallow(<ErrorScreen />);
    expect(errorScreen).toMatchSnapshot();
  });
});
