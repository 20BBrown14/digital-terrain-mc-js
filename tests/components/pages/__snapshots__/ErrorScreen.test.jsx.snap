// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorScreen renders a default view 1`] = `
<div
  className="error-screen-container"
>
  <div
    className="error-screen-element"
  >
    <img
      alt=""
      src="test-file-stub"
    />
    <Title>
      Something happened trying to load this page. Please try again later.
    </Title>
  </div>
</div>
`;
